config/metadata.csv

```diff
@@ -0,0 +1 @@
+
```

Comment:
Would you be able to add `/government/publications/test-business-readiness,yes` here so we can test the whole journey on integration with this piece of content?